src/criros/raveutils.py

```diff
@@ -94,6 +94,9 @@ def __init__(self, envid, fixed_frame='base_link', rate=10., logger=TextColors()
     # Start a thread to update all the objects found in TF and OpenRAVE
     self.rate = rate
     self.elapsed_time = 0.0
+    self.update_rave_environment() # Update env at least once in case stop() 
+                                   # is called immediately without delay
+
```

Comment:
Remove extra comment and empty line. Just keep it like this:
```{python}
self.elapsed_time = 0.0
self.update_rave_environment() # Update env at least once in case stop() 
self.timer = rospy.Timer(rospy.Duration(1.0/rate), self.update_rave_environment)
```